app/assets/javascripts/components/school_search.js.erb

```diff
@@ -126,14 +126,8 @@ document.addEventListener("DOMContentLoaded", function () {
 
       schoolIdInput.value = school.id;
 
-      if (currentPolicy == "additional-payments") {
-        methodInput.value = "patch";
-        setSubmitValue(defaultFormSubmitValue);
-      } else {
-        form.method = "post";
-        methodInput.value = "patch";
-        setSubmitValue("Continue");
-      }
+      methodInput.value = "patch";
```

Comment:
I think the `currentPolicy` variable is now not used anywhere and can be removed. Thanks for consolidating this.